### ISSUE TITLE
Broken with uvloop 0.13.0 and 0.14.0

### ISSUE THREAD

**Issue by talwrii** [vxgmichel/aioconsole#62]
Works with uvloop 0.12.0 however.

Reproducible code:

```
import asyncio
import uvloop
import aioconsole

uvloop.install()
loop = asyncio.get_event_loop()
loop.run_until_complete(aioconsole.interact())
```
```
Exception: ```Python 3.7.7 (default, Mar 10 2020, 15:43:03) 
[Clang 11.0.0 (clang-1100.0.33.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
---
This console is running in an asyncio event loop.
It allows you to wait for coroutines using the 'await' syntax.
Try: await asyncio.sleep(1, result=3)
---
>>> Assertion failed: (loop->watchers[w->fd] == w), function uv__io_stop, file src/unix/core.c, line 929.```

**Comment by vxgmichel** [comment]
Hi @talwrii and thanks for your report :)

I have been able to reproduce the problem on my machine. I ran a quick investigation but I haven't been able to find a quick workaround. This would require further investigation but I don't have time to do that at the moment. 

If anyone has any idea about what's going on I'd be happy to hear about it.

I hope it's not too much of a problem for you @talwrii, I'll let you know when I'll get back to it.
